### DISCUSSION TITLE
Access clickhouse with Tabix

### DISCUSSION THREAD

**Discussion by FV-Valraiso** [plausible/analytics#662] [Self-Hosted Support]
Hi,

I try to access clickhouse data using Tabix.
Clickhouse runs on port 8123
![Capture d’écran 2021-01-28 à 15 52 10](https://user-images.githubusercontent.com/1353509/106155712-40c10880-6181-11eb-8f97-fb2c1c98c7a5.jpg)

I tried using no user/password and using user "default" without password but I can't access clickhouse.

![Capture d’écran 2021-01-28 à 15 50 31](https://user-images.githubusercontent.com/1353509/106155555-196a3b80-6181-11eb-8872-3ea90401d1e5.jpg)

I also search in the code, but didn't find anything for a clickhouse password.

**Comment by FV-Valraiso** [comment]
No one as an idea ? :-(

**Reply by ukutaht** [reply]
No idea, I've never used Tabix

**Comment by Vigasaurus** [comment]
I'm not entirely sure about Tabix - but with my development instance of Clickhouse (similarly username and password-less), my DBMS connects with the following settings - no specified username or password. I think Clickhouse infers a username of `default` if none is provided. Maybe that'll work for Tabix? If you try the same with `clickhouse-client` in the CLI, does that connect?

![image](https://user-images.githubusercontent.com/19434157/109751147-79e70100-7ba3-11eb-9da2-952b75beedad.png)

**Reply by FV-Valraiso** [reply]
Thanks for reply
It doesn't help me in my case.
I have to look in clickhouse-server config.xml

I'll post if I find a solution
